mobile/examples/image_classifications/android/README.md

```diff
@@ -1,46 +1,42 @@
 # ONNX Runtime Mobile image classification Android sample application
 
 ## Overview
-This is an example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android. The demo app uses image classification which is able to continuously classify the objects it sees from the device's camera in real-time and displays the most probable inference result on the screen.
+This is an example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android. The demo app uses image classification which is able to continuously classifying the objects it sees from the device's camera in real-time and displays the most probable inference results on the screen.
 
 This example is loosely based on [Google CodeLabs - Getting Started with CameraX](https://codelabs.developers.google.com/codelabs/camerax-getting-started)
 
 ### Model
-We use classic MobileNetV2(float) model and MobileNetV2 (uint8) in this sample app.
+We use pre-trained [TorchVision MOBILENET V2](https://pytorch.org/hub/pytorch_vision_mobilenet_v2/) in this sample app.
 
 ## Requirements
 - Android Studio 4.1+ (installed on Mac/Windows/Linux)
 - Android SDK 29+
 - Android NDK r21+
-- Android device in developer mode and enable USB debugging
+- Android device with a camera in [developer mode](https://developer.android.com/studio/debug/dev-options) with USB debugging enabled
 
 ## Build And Run
-### Prerequisites
--  MobileNetV2 ort format model
--  labels text file (used for image classification)
--  Prebuilt ONNX Runtime arm64 Android Archive(AAR) files, which can be directly imported in Android Studio
 
-The above three files are provided and can be downloaded [here](https://1drv.ms/u/s!Auaxv_56eyubgQX-S_kTP0AP66Km?e=e8YMX1).
-
-[Optional] You can also build your own ONNX Runtime arm64 AAR files for Android. (See [build instructions here](https://www.onnxruntime.ai/docs/how-to/build.html#android) and [Build Android Archive(AAR)](https://www.onnxruntime.ai/docs/how-to/build.html#build-android-archive-aar)).
+### Step 0. [Optional] Prepare the ORT models
+Open [Mobilenet v2 Quantization with ONNX Runtime Notebook](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/notebooks/imagenet_v2/mobilenet.ipynb), this notebook will demostrate how to,
+1. Export the pre-trained MobileNet V2 FP32 model from PyTorch to a FP32 ONNX model
+2. Quantize the FP32 ONNX model to an uint8 ONNX model
+3. Convert both FP32 and uint8 ONNX models to ORT models
 
+Note: this step is optional, you can download the FP32 and uint8 ORT models [here](https://1drv.ms/u/s!Auaxv_56eyubgQlfGIBWh-j1wYyl?e=GYKoL7).
 
```

Comment:
If it's easy enough to just run the notebook I don't think we should provide a download as the ORT models may differ between ORT versions in the future.

Reply:
The quantized model from the notebook is quantized using some example images, not based on a proprietary calibration set, so the model will have accuracy issue.
This note book is showing how to export pytorch model -> quantize -> convert to ort, but will not generate a working quantized model.

Reply:
OK. For now it's ok but we may want to point to a landing page in the docs where they can download models for a particular ORT version (e.g. we add new optimizers or contrib ops or the ORT model format version changes for some reason).